source/manual/2nd-line-drills.html.md

```diff
@@ -92,3 +93,48 @@ Either before merging the change, or part way through the continuous deployment
 Follow the deployment pipeline in Jenkins. Confirm that no further environment deployments are triggered. For example, if you implemented the deploy freeze just after the app was deployed to Staging, confirm that the app was then not automatically deployed to Production.
 
 Remove the code freeze, then manually push the changes to all remaining environments so that they're in sync.
+
+## Drill getting a password out of govuk-secrets
+
+Most of our shared accounts are stored in this repo. Follow the steps to [get a password](https://github.com/alphagov/govuk-secrets/tree/main/pass#get-a-password), a useful one for 2nd line is the [Technical 2nd line Dashboard](https://alphagov.github.io/frame-splits/index.html?title=2nd+Line+Dashboard&layout=2x1-responsive&url%5B%5D=https%3A%2F%2Fgovuk-secondline-blinken.herokuapp.com%2Fblinken.html&url%5B%5D=https%3A%2F%2Fgrafana.production.govuk.digital%2Fdashboard%2Ffile%2F2ndline_health.json&url%5B%5D=https%3A%2F%2Fgovuk-zendesk-display-screen.herokuapp.com&url%5B%5D=) which shows a high level overview of the state of GOV.UK environments.
+Upon loading this dashboard it will ask for the `govuk-zendesk-display-screen` credentials.
+
+## Drill assigning a user to their publisher in data.gov.uk
+
+[Log into our shared data.gov.uk publisher account](/manual/data-gov-uk-2nd-line.html#logging-into-the-publisher). Pick a [publisher](https://ckan.publishing.service.gov.uk/organization) to do a hypothetical walk though of [Assign users to publishers](/manual/data-gov-uk-2nd-line.html#assign-users-to-publishers-setting-user-permissions).
```

Comment:
This is something we may have to do on 2nd line occasionally, but I wouldn't ever expect to come up as an on-call task (the rough barometer of whether or not we should be drilling something; in-hours tasks don't need it as much because there's enough developer support available during the day).

I'd be tempted to either remove this or combine it with the "changing user's permissions in Signon" task, creating a "Drill making changes to user accounts" 'super task'.